tests/unit/src/Internal/Payment/State/ProcessedStateTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Class ProcessedStateTestTest
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Tests\Internal\Payment\State;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use WC_Helper_Intention;
+use WCPay\Internal\Payment\PaymentContext;
+use WCPay\Internal\Payment\State\CompletedState;
+use WCPay\Internal\Payment\State\ProcessedState;
+use WCPay\Internal\Payment\State\StateFactory;
+use WCPay\Internal\Service\OrderService;
+use WCPAY_UnitTestCase;
+
+/**
+ * Tests for the initial payment state.
+ */
+class ProcessedStateTest extends WCPAY_UnitTestCase {
+	/**
+	 * State under test.
+	 *
+	 * @var ProcessedState
+	 */
+	private $sut;
+
+	/**
+	 * @var StateFactory|MockObject
+	 */
+	private $mock_state_factory;
+
+	/**
+	 * @var OrderService|MockObject
+	 */
+	private $mock_order_service;
+
+	/**
+	 * Set up the test.
+	 */
+	protected function setUp(): void {
+		parent::setUp();
+
+		$this->mock_state_factory = $this->createMock( StateFactory::class );
+		$this->mock_order_service = $this->createMock( OrderService::class );
+		$this->mock_context       = $this->createMock( PaymentContext::class );
+
+		$this->sut = new ProcessedState(
+			$this->mock_state_factory,
+			$this->mock_order_service
+		);
+		$this->sut->set_context( $this->mock_context );
+	}
+
+	public function test_complete_processing_will_transition_to_completed_state() {
+
```

Comment:
RadoCS: Let's remove this empty line.

Reply:
FIxed here: https://github.com/Automattic/woocommerce-payments/pull/7471/commits/48da0fc92461c2e53787cf45c2e79dcf0fa5bf8d